project.py

```diff
@@ -266,10 +266,50 @@ def get_credit_card_type(text: str) -> dict:
             raise ValueError(f'Invalid credit card number: {t.strip()}')
     return results
 
+def area_code_prefixes() -> list:
+    """Returns a list of area code prefixes that do not belong to the US and its territories"""
+
+    state_names = 'Alabama, Alaska, Arizona, Arkansas, California,  Colorado, Connecticut, Delaware, Florida, Georgia, Hawaii, Idaho, Illinois, Indiana, Iowa, Kansas, Kentucky, Louisiana, Maine, Maryland, Massachusetts, Michigan, Minnesota, Mississippi, Missouri, Montana, Nebraska, Nevada, New Hampshire, New Jersey, New Mexico, New York, North Carolina, North Dakota, Ohio, Oklahoma, Oregon, Pennsylvania, Rhode Island, South Carolina, South Dakota, Tennessee, Texas, Utah, Vermont, Virginia, Washington, West Virginia, Wisconsin, Wyoming, District of Columbia, American Samoa, Guam, Northern Mariana Islands, Puerto Rico, United States Minor Outlying Islands, U.S. Virgin Islands'

```

Comment:
You could restructure get_state_abbrev_freq() to avoid needing a duplicate of all the state names.  You could write a new function called get_state_abbrev() which returns a list of states and then call that new function in get_state_abbrev_freq() and also your new area_code_prefixes()
```
def get_state_abbrev_freq(text_states: str) -> dict:
    state_to_abb_dict = get_state_abbrev()
    
    states = [x.strip() for x in text_states.split(",")]
    state_freq_dict = {}
```

Reply:
Regarding the restructuration of the function get_state_abbrev_freq(text_states: str) -> dict even if we write another function, get_state_abbrev_freq(text_states: str) still needs a parameter passed to it, a string of state names.

Reply:
see #32 for what I was thinking

Reply:
I saw it,thanks. I will approve and merge it.